misk/src/main/kotlin/misk/clustering/weights/ActiveClusterWeight.kt

```diff
@@ -0,0 +1,22 @@
+package misk.clustering.weights
+
+import misk.inject.KAbstractModule
+
+/**
+ * A static [ClusterWeightProvider] that always returns 100
+ */
+class ActiveClusterWeight : ClusterWeightProvider {
+
+  override fun get(): Int {
```

Comment:
don’t love Integers for this, though I also don’t love floats?

Reply:
i'm open to suggestions, but not really a fan of floats. is the float value 0.801 or 80.1? at least an Int is clear in that regard. Loss of precision didn't seem important for something like cluster weights.

I briefly considered a boolean, since in practice it's 0 or 100. However, it's reasonable in the future we might support a canary cluster running some percentage of the background tasks.